apps/meteor/tests/e2e/account-forgetSessionOnWindowClose.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { DEFAULT_USER_CREDENTIALS } from './config/constants';
+import { Registration } from './page-objects';
+import { test, expect } from './utils/test';
+
+test.describe.serial('Forget session on window close setting', () => {
+	let poRegistration: Registration;
+
+	test.beforeEach(async ({ page }) => {
+		poRegistration = new Registration(page);
+
+		await page.goto('/home');
+	});
+
+	test.describe('Setting off', async () => {
+		test.beforeAll(async ({ api }) => {
+			await api.post('/settings/Accounts_ForgetUserSessionOnWindowClose', { value: false });
+		});
+
+		test('Login using credentials and reload to stay logged in', async ({ page, context }) => {
+			await poRegistration.username.type('user1');
+			await poRegistration.inputPassword.type(DEFAULT_USER_CREDENTIALS.password);
+			await poRegistration.btnLogin.click();
+
+			await expect(page.locator('role=heading[name="Welcome to Rocket.Chat"]')).toBeVisible();
+
+			const newPage = await context.newPage();
```

Comment:
Doesn't it closes the entire context (window) after the test is done?

Reply:
I think, if we create a new page using `browser.newPage()` that would open a new browser window, which would need closing as you mentioned.

But if we use `context.newPage` as used in test here, it would open a new tab in same window. This window is closed on it's own after each test, so we don't need to close it separately.

Reply:
ouch, that's correct man, my bad. You are right.